src/main/java/com/example/githubapi/Config/MyErrorDecoder.java

```diff
@@ -0,0 +1,35 @@
+package com.example.githubapi.Config;
+
+import com.example.githubapi.Exceptions.NotAcceptableException;
+import com.example.githubapi.Exceptions.UserNotFoundExceptions;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.FeignException;
+import feign.Response;
+import feign.RetryableException;
+import feign.codec.ErrorDecoder;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class MyErrorDecoder implements ErrorDecoder {
```

Comment:
Mamy w Springu cos co nazywamy filtrami. 
https://www.baeldung.com/spring-boot-add-filter
W filtrach mozemy np weryfikowac przychodzace requesty.
Sprobujmy wiec stworzyc taki filter i zrobic w nim walidacje czy request przychodzacy nie zawiera Headeru 
Accept o wartosci application/xml
Jesli zawiera to zwrocmy wlasnie nasz ExceptionMessage z odpowiednim napisem.

Reply:
OncePerRequestFilter <- do poczytania